stellargraph/datasets/__init__.py

```diff
@@ -19,4 +19,19 @@
 
 """
 
+__all__ = [
```

Comment:
It's a little bit of a pest that we have to duplicate the list of definitions here. Could you say why it was needed just so I understand (and potentially add a comment, if it's subtle)?

Reply:
It seemed like `:members:` in `api.txt` looks for `__all__` - the docs for this section came out empty without this

Reply:
I've added a comment